### DISCUSSION TITLE
the corner_radius argument of CTkLabel

### DISCUSSION THREAD

**Discussion by leaziat** [TomSchimansky/CustomTkinter#1947] [General]
whatever value I set, the angle remains at 90 degrees

**Comment by Akascape** [comment]
@leaziat 
All customtkinter widgets are rectangular in shape, the round corner is just a drawing trick in canvas that camouflages the bg_color.

![Example](https://user-images.githubusercontent.com/89206401/189685084-f3bacda3-88c7-415c-93e5-916c61c2c1c0.png)

By default the label has same bg and fg color and that is why the corner radius is not visible. You first have to change the fg color if you want the round corners.

Secondly, if you are trying to get a transparent background below it then it is not possible.
